backend/src/main/java/com/google/sps/data/Participant.java

```diff
@@ -32,6 +32,8 @@
   private final String role;
   /** Product area at Google */
   private final String productArea;
+  /** List of interests delimited by commas */
+  private final String interests;
```

Comment:
The comma separated String is a good storage format but a List of Strings would be easier to work with in Java. Also converting between List<String> and comma-separated String in participant datastore prevents the rest of the code from depending on an implementation detail of the storage

Reply:
Added a ListStringConversion class with static methods to easily convert between List for Participant/User and String for their datastore representations!